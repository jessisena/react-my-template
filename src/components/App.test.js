import React from "react";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";
import App from "./App";


describe("App component", () => {

	const component = shallow(<App />);

	it("renders and has only 1 div element", () => {

		expect(component.find("div")).toHaveLength(1);
	
	});
  
	it("renders correctly", () => {

		expect(toJSON(component)).toMatchSnapshot();
	
	});

});
