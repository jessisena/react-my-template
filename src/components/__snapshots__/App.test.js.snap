// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders correctly 1`] = `
<div
  className="App"
>
  <h1>
    TEMPLATE REACT APP
  </h1>
</div>
`;
